src/components/Summary/DocumentsDelivery.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <section id="document-delivery-section">
+    <header>
+      <h2>X. Alteration Documents Delivery</h2>
```

Comment:
Looks like "X" was a placeholder in the designs.

I don't know if you can set an actual section number here, or whether something more complex is needed to handle conditional sections.

What's your plan?

Reply:
I made it a 1 for now